### ISSUE TITLE
CS1: Check if GoG installer version is identical to Steam build.

### ISSUE THREAD

**Issue by AdmiralCurtiss** [AdmiralCurtiss/SenPatcher#31]
I thought it was but I'm getting a conflicting report on the Steam community forums about it (for CS1).

**Comment by AdmiralCurtiss** [comment]
(check CS2 too while we're here)

**Comment by AdmiralCurtiss** [comment]
Another report of this, with GoG Galaxy this time. Something odd is going on here.

**Comment by AdmiralCurtiss** [comment]
Ah.

~~~
0dee9f419797624924558dc3f8afeb0ae877580c *data\effects\samples\daeéęéšâGâtâF.eff
cc1b4d4640ab06bc6525ba2d1e88e86beac11ef0 *data\effects\samples\âpâ^ü[âôâAâjâü.eff
c9b746303dbf8795dcfac5591d97c9fab46b7678 *data\effects\samples\âXâeâőâXĽŚ.eff
4ef7483f44589763748f3b5f48bb6809a330f582 *data\effects\samples\â^ü[âQâbâgéęéš.eff
c4fdbc028659a2c8048dbf0fc0a567b3cb52e888 *data\effects\samples\Äęô]é-î÷ô]ôKŚp.eff
986a81e94b981282a8629871e99a5999442ab071 *data\effects\samples\ë~ĺî(ĆŃë¦Ć+éŽ).eff
462eeaf7ee00155dc2851d0d2ac2dad93d90241c *data\effects\samples\ö-ë~ĺî.eff
35eaa8c4619455d40b49d29db60de6aa8dd2b7d9 *data\effects\samples\ĆdŚ=(ĆŇôT).eff
b10361f42574696ae0b7983bebded70e04d52a52 *data\effects\samples\ĆdŚ=(ĺÁéTĽďéŔ).eff
f4097d4f84c73c1631272535cdbdc49364d99ece *data\effects\samples\łţŹL.eff
fcca406cdc4db09f3a9f11a61db10b2f51fe12e2 *data\effects\samples\őOÉŇ(Śv-L»+).eff
ed02bbc1a9bbe0d7bac82dd571d7ea86c66c65b1 *data\effects\samples\őOÉŇ.eff
04225404e9334b1e12d03a71841f5d1689ad10db *data\effects\samples\őů(ĆŃë¦Ć+éŽ).eff
638547eb41c7ac3c9e3e577fbc3f629243899e59 *data\effects\samples\őů.eff
5f6e1c9d3a56c87be91e3d55b0bc2849ccd41bae *data\effects\samples\Ć\ÄÜ.eff
684cc74b0837ff1408124f8b8a05cfd9c9a09195 *data\se\wav\ed8éŹ2123.wav
~~~

Looks like someone broke the encoding on the filenames on GoG.

**Comment by AdmiralCurtiss** [comment]
CS2 seems to be fine, both Steam/GoG versions have proper encoding.

**Comment by AdmiralCurtiss** [comment]
CS2 Humble has wrong encoding, pfft. Okay, fine, I'll run a fixup step before opening the game folder...

**Comment by AdmiralCurtiss** [comment]
Fixed in v0.6, probably.
